doc/index.rst

```diff
@@ -204,6 +204,7 @@ Note that some of these currently tied only to a single option, but we still lea
 * **ROS_REPO** (default: ros-shadow-fixed): `ROS_REPO` can be used to set `ROS_REPOSITORY_PATH` based on known aliases: 'ros`/`main`, 'ros-shadow-fixed`/`testing` or `building`.
 * **ROS_REPOSITORY_PATH**: Location of ROS' binary repositories where depended packages get installed from (typically both standard repo (`http://packages.ros.org/ros/ubuntu`) and `"Shadow-Fixed" repository <http://wiki.ros.org/ShadowRepository>`_ (`http://packages.ros.org/ros-shadow-fixed/ubuntu`)). Since version 0.3.4, `ROS_REPO` is recommended, and `ROS_REPOSITORY_PATH` is for more intermediate usage only (e.g. to specify your own binary repository (non-standard / in house)). Backward compatibility is preserved.
 * **ROSDEP_SKIP_KEYS** (default: not set): space-separated list of keys that should get skipped by `rosdep install`.
+* **ROSINSTALL_AUTOMERGE** (default: true): When set to true, any `wstool merge` operations will have the `-y` option set, preventing any CLI prompts from appearing when any element details are changed. Set this to false in order to have visibility on element detail changes. WARNING: If this is set to false, and a prompt occurs due to detail changes, the running CI process will halt indefinitely.
```

Comment:
+1. Thorough description.

Just as FYI; my preference, after the merge into this forked repo, is to send this change to the upstream [ros-industrial/industrial_ci](https://github.com/ros-industrial/industrial_ci) repo as this is generic feature. One  possible feedback from the upstream review would be to generalize the arguments so that a user can pass any argument `wstool` takes in, not just `-y`. But there can be also a drawback in doing that, and I don't have an answer for what's the best, so I think the current state is fine for now.

Reply:
>One possible feedback from the upstream review would be to generalize the arguments so that a user can pass any argument wstool takes in, not just -y.

Right, I actually had this same thought while I was implementing it and decided not to venture down that rabbit hole since the potential for side effects could be pretty large, but I may have to do it after all. Thanks for the head's up